### PR TITLE
Add nerd-fonts via single package.json w/ git auto-update

### LINKED ISSUE #12195: [Request] Add nerd-fonts
**Library name:** nerd-fonts
**Library description:** 🔡 Iconic font aggregator, collection, and patcher: 39 patched fonts (87k+ variations). Includes popular glyph collections such as Font Awesome & fonts such as Hack
**Git repository url:** https://github.com/ryanoasis/nerd-fonts
**npm package name or url** (if there is one): 
**License (List them all if it's multiple):** MIT
**Official homepage:** https://nerdfonts.com/

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/51592418-request-add-nerd-fonts?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F32893&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by MattIPv4**
Resolves #12195

### CODE REVIEW

File: ajax/libs/nerd-fonts/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "url": "git://github.com/ryanoasis/nerd-fonts.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ryanoasis/nerd-fonts.git",
+    "fileMap": [
+      {
+        "basePath": "patched-fonts",
+        "files": [
+          "**/*"
```
Comment (sashberd) [inline ajax/libs/nerd-fonts/package.json:30]: There are a lot of  *.md files, some *.ps1 files. Do we need them?

Comment (MattIPv4) [reply ajax/libs/nerd-fonts/package.json:30]: The md files give useful context on each of the fonts (source, licenseing, etc.) so whilst they won't be available on the CDN they're useful to keep in the repo. I'm not sure what the ps1 files are, but I think its probably safer to just mirror this entire dir.

Comment (sashberd) [reply ajax/libs/nerd-fonts/package.json:30]: For your discretion Matt, however, I personally never used for md file for any imports in apps. For information yes, but not in code

Comment (MattIPv4) [reply ajax/libs/nerd-fonts/package.json:30]: Yeah, they won't even be available on the CDN, so you couldn't use them. I just think they should be copied across into our repo to give context on all these font files as it's a big library.

**Pull request merged**